core/touch.js

```diff
@@ -113,6 +113,7 @@ Blockly.longStop_ = function() {
  * @private
  */
 Blockly.onMouseUp_ = function(e) {
+  console.log('on mouse up');
```

Comment:
Deliberately left in for now, since this branch (dragging) isn't ready to merge to develop yet, and I'm using this to make sure that I'm ordering mouse event handling correctly.

I think if I do it right this entire function disappears.